.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 3
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 4
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
```

Comment:
this issue or this pull request?

Reply:
Both really. Configuration will apply to both issues and pull requests. That comment is really copied from example documentation published by bot team.

I am not sure why they say 'issue', maybe because usually PR is associated with an issue. On the other hand it does not make much difference IMO.